versions.tf

```diff
@@ -1,12 +1,10 @@
 terraform {
-  required_version = ">= 0.14.11"
+  required_version = ">= 0.15.00"
```

Comment:
Why is this bumped?

Reply:
Presumption that the world should move on 😆 
Honestly 15 was when most stuff became stableish and I couldn't find a simple "AWS Provider 4 supports version x above" in the repo'; then I remembered its burried in the use-provider option 🙄 
![image](https://user-images.githubusercontent.com/83597/223996774-b5859097-894b-4eb9-b1f2-63f250f560c9.png)